types/Building.ts

```diff
@@ -0,0 +1,37 @@
+import { Building, BuildingEntry } from '@prisma/client';
+
+export type BuildingDO = Building & { building_entries: BuildingEntry[] };
```

Comment:
Reorder the definitions to have 3 of them regarding the same object next to each other, don't interweave them